server/src/main/java/org/eclipse/hono/registration/RegistrationMessageFilter.java

```diff
@@ -64,14 +65,33 @@ public static boolean verify(final ResourceIdentifier linkTarget, final Message
          } else {
              return true;
          }
-    }
+     }
 
-     private static boolean hasValidAction(final Message msg) {
-         String action = MessageHelper.getApplicationProperty(
+     private static String getAction(final Message msg) {
```

Comment:
I think we should make `RegistrationConstants.getAction()` public and remove this method.